unit-tests-jest/numbers.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers) {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount) {
+  return `$${amount}.00`;
```

Comment:
Write a test that formats a fractional number. That will expose the bug in this function.